### PR TITLE
NIAD-1482: Add gpc 1.5.1 to docker-compose

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

**Comment by mattd-kainos** [comment]
Consider adding a vars.sh example for running the adaptor against the mock?

**Comment by Broom94** [comment]
There are no additional variables. This is only to unblock GPC work for this sprint

**Review by matthewdoreilly — COMMENTED** [review]
Do we still need the `gpcc` service in docker-compose with this new Docker image being added?

**Comment by Broom94** [comment]
yes we do, the new docker images provided is to replace the wiremock / public demonstrator requests from gpcc

**Review by ElliotB-96 — DISMISSED** [review]
Looks good to me, able to run locally

**Review by msolecka — APPROVED** [review]

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -93,3 +93,54 @@ services:
...
+    environment:
+      - TZ=Europe/London
+    links:
+      - gpconnect-api
+    depends_on:
+      - gpconnect-api
+    
+networks: 
+  gpconnect-network-1.5:
+    driver: bridge
```
Comment (lgonczar) [inline docker/docker-compose.yml:146]: Should we use the same network for GPC services as for GP2GP? Not sure if the containers are going to be able to talk to each other.

**Pull request closed without merging**